### PR TITLE
Feature/default login route

### PR DESCRIPTION
**Pull request by amolthiteapptware**
The default page will be sign-in. If the user Is not logged in
"back to simppple" on the sign-in page removed
NFT marketplace removed from left side

### CODE REVIEW

**Comment by amolthiteapptware** [comment]
@rishibhargava please check this Pull request once,  I am not able to add the Reviewers

File: src/views/auth/signIn/index.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default function DescopeSignIn() {
 					// theme={ colorMode }
 					flowId={process.env.REACT_APP_DESCOPE_SIGN_IN_FLOW_ID || "sign-in"}
 					onSuccess={(e) => {
-						console.log("Success!" + JSON.stringify(e.detail.user));
+						localStorage.setItem('logedInUserDetails', JSON.stringify(e.detail.user))
```
Comment (rishibhargava) [inline src/views/auth/signIn/index.tsx:36]: We should not save user in localStorage. We already get the user whenever we want from descope hooks. It is called user and you can do const { authenticated, user } = useAuth();

File: src/index.tsx [hunk 2]
```diff
@@ -26,7 +26,12 @@ const App = () => {
             <Route path={`/auth`} component={AuthLayout} />
             <Route path={`/admin`} component={AdminLayout} />
             <Route path={`/rtl`} component={RTLLayout} />
-            <Redirect from="/" to="/admin" />
```
Comment (rishibhargava) [inline src/index.tsx:29]: use authenticated from useAuth here rather than localStorage.

File: src/layouts/auth/Default.tsx [hunk 3]
```diff
@@ -28,7 +28,7 @@ function AuthIllustration(props: { children: JSX.Element | string; illustrationB
 				ps={{ xl: '70px' }}
 				justifyContent='start'
 				direction='column'>
-				<NavLink
+				{/* <NavLink
```
Comment (rishibhargava) [inline src/layouts/auth/Default.tsx:31]: I dont like leaving commented code. If we dont need this, lets remove this completely.

**Review by rishibhargava — APPROVED** [review]
Approved after you make those changes.

File: src/index.tsx (hunk 2 above)
Comment (amolthiteapptware) [reply src/index.tsx:29]: use of Local storage removed

**Review by rishibhargava — COMMENTED** [review]
Looks great. Did not test. So if it works, please merge.

**Comment by dorsha** [comment]
@amolthiteapptware @rishibhargava this PR cause a regression and endless loop when providing wrong project id, we should have proper testing here.

File: src/layouts/admin/index.tsx [hunk 4]
```diff
@@ -5,12 +5,21 @@ import Footer from 'components/footer/FooterAdmin';
...
+import { useAuth } from "@descope/react-sdk";
+import { useHistory } from "react-router-dom";
 
 // Custom Chakra theme
 export default function Dashboard(props: { [x: string]: any }) {
+	const { authenticated } = useAuth();
+	let history = useHistory();
+	useEffect(() => {
+		if (!authenticated) {
+			history.push("/auth");
```
Comment (dorsha) [inline src/layouts/admin/index.tsx:20]: this causes infinite loop.

File: src/layouts/auth/index.tsx [hunk 5]
```diff
@@ -12,6 +14,14 @@ import { SidebarContext } from 'contexts/SidebarContext';
 export default function Auth() {
 	// states and functions
 	const [ toggleSidebar, setToggleSidebar ] = useState(false); 
+	const { authenticated } = useAuth();
+	let history = useHistory();
+	
+	useEffect(() => {
+		if (authenticated) {
+			history.push("/admin");
```
Comment (dorsha) [inline src/layouts/auth/index.tsx:22]: and this

**Comment by rishibhargava** [comment]
@asafshen Can you help us fix this? Amol is out on Monday for Christmas holiday in India.

**Comment by asafshen** [comment]
> @asafshen Can you help us fix this? Amol is out on Monday for Christmas holiday in India. 

Yes, will fix today 👍

**Pull request merged**